docs/cpp/configure-intellisense.md

```diff
@@ -0,0 +1,121 @@
+ContentId: 381b7ce1-5766-49b0-ad26-f9eedae70e63
+PageTitle: Configure IntelliSense
+DateApproved: 8/23/2023
+MetaDescription: Configure Visual Studio Code IntelliSense in the C/C++ Extension
+
+---
+# Configuring IntelliSense
+
+This article is about configuring the C/C++ extension to provide proper [IntelliSense](/docs/editor/intellisense.md). IntelliSense is a helpful tool built into VS Code that provides various code editing features that help you code faster and more efficiently. For example, code completion, parameter info, syntax highlighting, code actions (light bulbs), and member lists are all generated using IntelliSense.
+
+IntelliSense is powered by a language service, which provides C++ specific language completions in your code. For the extension to provide correct IntelliSense suggestions, you need to configure the C/C++ extension. There are three types of sources that can be used for configuration:
+
+1. A C/C++ compiler
+1. compile_commands.json file
+1. Another extension (for example, the [Makefile](https://marketplace.visualstudio.com/items?itemName=ms-vscode.makefile-tools) or [CMake extension](/docs/cpp/CMake-linux.md)).
+
+## When is IntelliSense automatically configured?
+
+The C/C++ Extension, in most occasions, configures IntelliSense automatically for you. Common compiler paths in world writeable locations are queried on your machine, and if a compiler is available, are configured. If a compiler in a non-world writeable location is determined, a notification is displayed asking to configure this compiler. When approving compilers for IntelliSense, ensure that the compiler path is correct. Common compilers in their default world writeable locations that are automatically configured include the clang compiler, GCC, MinGW, and MSVC. If a compile_commands.json file or another extension that provides a configuration is found in your workspace, the extension will also automatically use these options to configure IntelliSense.
+
+## How to check whether IntelliSense is configured
+
+There are multiple options to determine your IntelliSense status:
+
+### Option 1: Check for a status bar indicator
+
+ If you don't have IntelliSense configured, the C/C++ Extension shows an indicator in the status bar with a warning sign stating "Configure IntelliSense".
+
+![configure-intelliSense-indicator-status-bar](images/intellisense/configure-intellisense-indicator.png)
+
+To configure, select the status bar indicator, which navigates you to the [configuration quickpickpick](/docs/cpp/configure-intellisense#_selecting-a-compiler-for-IntelliSense.md). The status bar indicator also is shown if you have provided a configuration for IntelliSense, but the C/C++ Extension is reverting to the base configuration provided in your `c_cpp_properties.json` file. In this case, open the `c_cpp_properties.json` file (Select **C/C++: Edit Configurations (UI)** from the Command Palette (`kb(workbench.action.showCommands)`)).
```

Comment:
format should be (/docs/cpp/configure-intellisense.md#selecting-a-compiler-for-intellisense)
but I'm not sure that header exists

Reply:
Updated all the places this header was referenced - it is supposed to link to the configuration quickpick section, which I had renamed since creating the links.